web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.scss

```diff
@@ -37,9 +37,26 @@
     top: 12px;
     color: #555;
   }
+
+  &__workbasket-icon {
+    width: 16px;
+    height: 16px;
+    fill: #555;
+  }
+
+  &__additional-toolbar {
```

Comment:
There already is a css class named `additional-toolbar` see line 17.

Reply:
Merged